features/step_definitions/user_steps.rb

```diff
@@ -267,4 +274,8 @@ def get_series_name(age, classname, name)
 Then /^I should get confirmation that I changed my username$/ do
   step(%{I should see "Your user name has been successfully updated."})
 end
- 
\ No newline at end of file
+ 
+Then /^the user "([^"]*)" is activated$/ do |login|
```

Comment:
My only suggestion would be to use "should be activated" rather than "is activated" since we're checking that something is true (like with "I should see 'some text'" or "one email should be sent")